api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Visits/VisitRequestDTO.java

```diff
@@ -24,4 +24,12 @@ public class VisitRequestDTO {
 /*    private int petId;
     private int practitionerId;*/
     private boolean status;
+
+    public VisitRequestDTO(LocalDateTime now, String description, String petId, String practitionerId) {
+        this.visitDate = now;
```

Comment:
[Nitpick] I know we are using LocalDateTime but we're not using it in the context of it being a real-time event log (regarding what the 'now' variable name you gave it)